priv/sites/zotonic_status/modules/mod_zotonic_site_management/support/zotonic_status_addsite.erl

```diff
@@ -24,11 +24,11 @@
 
 -include_lib("zotonic.hrl").
 
--spec addsite(binary(), list(), #context{}) -> ok | {error, binary()|string()}.
+-spec addsite(binary(), list(), #context{}) -> {ok, {Site :: binary(), Options :: list()}} | {error, Reason :: binary() | string()}.
```

Comment:
Line 27 is too long: -spec addsite(binary(), list(), #context{}) -> {ok, {Site :: binary(), Options :: list()}} | {error, Reason :: binary() | string()}..